doc/api/cli.md

```diff
@@ -2018,6 +2018,37 @@ On a machine with 2 GiB of memory, consider setting this to
 $ node --max-old-space-size=1536 index.js
 ```
 
+### `--max-semi-space-size=SIZE` (in megabytes)
+
+Sets the maximum
+[semi-space](https://www.memorymanagement.org/glossary/s.html#semi.space)
+size for V8's [scavenge](https://v8.dev/blog/orinoco-parallel-scavenger)
+garbage collector in MiB.
+Increasing the max size of a semi-space may bring
+throughput improvement for Node.js at the cost of more
+memory consumption.
+Since the [young generation](https://v8.dev/blog/orinoco-parallel-scavenger)
+size of V8 heap is
+[three times](https://chromium.googlesource.com/v8/v8.git/+/refs/heads/main/src/heap/heap.cc#328)
+the size of semi-space,
+so the memory consumption is tripled when semi-space size increased,
```

Comment:
It might be alarming to users as they would think that it's the heap size that's tripled.

```suggestion
an increase of 1 MiB to semi-space will triple and cause the heap size to increase by 3 MiB.
```

Reply:
I'd phrase it as saying the option applies to each of the three individual semi spaces.

(It's really just two but then you have to explain that it's N for the large object nursery and 2*N for the normal object nursery and that's just getting into the weeds.)

Apropos links to code: can you use stable links, i.e., links to a specific tag or commit instead of a branch name?